### DISCUSSION TITLE
V0.11.13 请问如何打开本地存储的内容？

### DISCUSSION THREAD

**Discussion by winstonzhang123** [obsidianzh/forum#79] [General综合]
新手刚刚使用，查询过官方的使用说明，但是没找到对应的操作方法。
系统是win10，在本地文件夹内选择相应的.md文档，选择用Obsidian打开，弹出对话框，选择“打开库文件夹”。但里面看不到相应的内容文件。
![屏幕截图 2021-04-11 074920](https://user-images.githubusercontent.com/82294093/114287619-d985d580-9a9a-11eb-8330-782bf664562c.png)
![2](https://user-images.githubusercontent.com/82294093/114287622-ddb1f300-9a9a-11eb-81bc-821fe7499b95.png)

**Comment by Rayme** [comment]
打开库文件夹时，选择的是一个文件夹，是不会显示文件夹里的文件的。你选择“读书笔记”这个文件夹，就能看到ob里左侧有所有的md文件。

**Comment by lillian-who** [comment]
你需要去B站看一下使用方法。操作起来很简单，但是不看操作流程的话其实不好解释。obsidian打开的是“库”，这个概念，也就是一个文件夹。它打开的不是某个markdown文件。你可以把它理解为文件夹式的文档管理。
